src/Api/Endpoints/ImportNotifications/UpdatedImportNotificationRequest.cs

```diff
@@ -21,5 +21,11 @@ internal sealed class UpdatedImportNotificationRequest
             + "the request will be invalid."
     )]
     [FromQuery(Name = "to")]
-    public DateTime? To { get; init; } = DateTime.Now;
+    public DateTime To { get; init; }
```

Comment:
Description field still mentions 24 hours. We should have docs reflective of the logic that is implemented